src/pages/index.js

```diff
@@ -63,27 +100,102 @@ function handleImageClick(cardLink, cardCaption) {
   imagePopup.open(cardLink, cardCaption);
 }
 
+function handleAvatarClick() {
+  editAvatarPopup.open();
+}
+
+function handleDeletePopup(cardId, evt) {
+  deleteVerifyPopup.open();
+  cardIdForDeletion = cardId;
+  cardToDelete = evt.target.closest(".card");
+}
+
+function removeLikeFromServer(cardId, likeButton) {
+  api
+    .removeLikeFromCard(cardId)
+    .then(likeButton.classList.remove("card__liked_active"))
+    .catch((err) => console.error(err));
+}
+
+function likeToServer(cardId, likeButton) {
+  api
+    .addLikeToCard(cardId)
+    .then(likeButton.classList.add("card__liked_active"))
```

Comment:
Now only class `Card` knows every **card selector**, so you need to encapsulate the likes logics inside its methods and call them in `index.js`. It’s a violation of **Single responsibility** principle